.actions/assistant.py

```diff
@@ -30,6 +30,10 @@
         "requirements/app/ui.txt",
         "requirements/app/cloud.txt",
     ),
+    "lite": (
+        "requirements/lite/base.txt",
+        "requirements/lite/strategies.txt",
```

Comment:
What would this do for strategies?

Reply:
`req = open(fname, encoding="utf-8").read().replace(">=", "==")` which is what `oldest` is supposed to do.

But it can only do so, if the requirement file is listed here.

Reply:
some are used in CPU testing still, right?